cmd/cmd.go

```diff
@@ -98,7 +99,7 @@ func CreateHandler(cmd *cobra.Command, args []string) error {
 		return nil
 	}
 
-	if err := client.Create(context.Background(), &request, fn); err != nil {
+	if err := client.Create(cmd.Context(), &request, fn); err != nil {
```

Comment:
I thought the original reason why we didn't reuse the context from cobra was that it was weird to reuse that w/ the http context?

Reply:
I haven't seen any issues with using cobra.Context

Reply:
Seems fine to me, but it's slightly weird to reuse the context from cobra to pass it to the client.

Reply:
It's not necessarily context from cobra.Command. cobra.Command is being passed in a context which can be retrieved with cmd.Context(). See https://github.com/jmorganca/ollama/blob/main/main.go#L11.

This allow the same context to be used by multiple command, specifically client requests, which is useful for shared data